cmd/cluster/support/post_test.go

```diff
@@ -0,0 +1,131 @@
+package support
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osdctl/internal/utils/globalflags"
+)
+
+func TestReadTemplate(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	testCases := []struct {
+		title       string
+		option      *postOptions
+		errExpected bool
+	}{
+		{
+			title: "No template",
+			option: &postOptions{
+				GlobalOptions: &globalflags.GlobalOptions{},
+			},
+			errExpected: true,
```

Comment:
I'm not familiar enough with these templates to know why they would fail. Can you leave a short comment explaining why this should fail? Can you also add a success case?